tests/integration/lifecycle/test_snap.py

```diff
@@ -91,24 +83,24 @@ def test_snap(self):
                 dedent(
                     """\
                 name: assemble
-                version: '1.0'
+                version: 1.0
                 summary: one line summary
                 description: a longer description
                 architectures:
                 - {}
                 confinement: strict
-                grade: devel
+                grade: stable
                 apps:
                   assemble-bin:
-                    command: snap/command-chain/snapcraft-runner $SNAP/command-assemble-bin.wrapper
+                    command: command-assemble-bin.wrapper
                   assemble-service:
-                    command: snap/command-chain/snapcraft-runner $SNAP/command-assemble-service.wrapper
+                    command: command-assemble-service.wrapper
                     daemon: simple
-                    stop-command: snap/command-chain/snapcraft-runner $SNAP/stop-command-assemble-service.wrapper
+                    stop-command: stop-command-assemble-service.wrapper
                   binary-wrapper-none:
                     command: subdir/binary3
                   binary2:
-                    command: snap/command-chain/snapcraft-runner $SNAP/command-binary2.wrapper
+                    command: command-binary2.wrapper
```

Comment:
Given that legacy has its own version of this, would it be worth making it use a base? Or are you thinking that should involve a move to spread?